DFC.ServiceTaxonomy.GraphSync/Orchestrators/DeleteOrchestrator.cs

```diff
@@ -75,6 +75,13 @@ public async Task<bool> Delete(ContentItem contentItem)
             (IAllowSync publishedAllowSync, IDeleteGraphSyncer? publishedDeleteGraphSyncer) = deleteGraphSyncers[0];
             (IAllowSync previewAllowSync, IDeleteGraphSyncer? previewDeleteGraphSyncer) = deleteGraphSyncers[1];
```

Comment:
Do we need to consider nulls?

Reply:
As far as I can see Orchestrator.GetDeleteGraphSyncerIfDeleteAllowed(...) will throw an exception if it fails so I don't see this being a problem.